pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -3,6 +3,9 @@
 , secureBoot ? false
 , httpSupport ? false
 , tpmSupport ? false
+, tlsSupport ? false
+, debug ? false
+, sourceDebug ? debug
```

Comment:
```suggestion
, sourceDebug ? false
```

Reply:
Usually, you'd expect that `debug` enables also source-level debugging in the binary, any reason to commit your suggestion?

Reply:
Sorry my brain was completely fried apparently.  In my head this was the literal string "debug". What you did makes perfect sense.

Reply:
No worries!